src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/MaintainSupplierPricingController.kt

```diff
@@ -283,15 +286,6 @@ class MaintainSupplierPricingController(
     if (existingPrice.amount == price) result.rejectValue("exceptionPrice", "Invalid price")
 
     if (result.hasErrors()) {
-      model.apply {
```

Comment:
Turns out this bit of code was not needed due to there being a redirect which in turn already prepares the state for the redirected page.